packages/shared/lib/contexts/governance/actions/registerParticipationEvents.ts

```diff
@@ -1,15 +1,25 @@
 import { ParticipationEventMap, ParticipationEventRegistrationOptions } from '@iota/wallet/out/types'
 
+import { get } from 'svelte/store'
+
 import { IAccountState } from '@core/account/interfaces'
 
 import { addProposalsFromParticipationEventMap } from './addProposalsFromParticipationEventMap'
+import { selectedAccount } from '@core/account'
 
 export async function registerParticipationEvents(
     registrationOptions: ParticipationEventRegistrationOptions,
     account: IAccountState
 ): Promise<ParticipationEventMap> {
-    const eventMap = await account.registerParticipationEvents(registrationOptions)
-    addProposalsFromParticipationEventMap(eventMap, account)
+    let newRegistrationOptions = registrationOptions
+    if (get(selectedAccount)?.removedProposalIds?.length > 0) {
+        newRegistrationOptions = {
+            ...registrationOptions,
+            eventsToIgnore: get(selectedAccount).removedProposalIds,
```

Comment:
We should make sure that an empty array is passed if this property is undefined, i.e.
```typescript
eventsToIgnore: get(selectedAccount)?.removedProposalIds ?? []
```